tools/topology/topology2/platform/intel/hdmi-in-ssp-config.conf

```diff
@@ -0,0 +1,56 @@
+#SSP DAI configuration to receive samples via I2S(in slave mode)
+#from HDMI input converter chip.
+Object.Dai.SSP [
+	{
+		id				$HDMI_IN_1_ID
+		dai_index		$HDMI_IN_1_INDEX
+		direction		"capture"
+		name			$HDMI_IN_1_NAME
+		default_hw_conf_id	0
+		sample_bits		32
+		io_clk			$HDMI_IN_MCLK
+		Object.Base.hw_config.1 {
+			id		0
+			name		"HDMI-In-1"
+			mclk_freq	$HDMI_IN_MCLK
```

Comment:
I am confused on this, usually this would result in an mclk being generated by the SOC, but it seems you are relying on an external MCLK?

Reply:
I understand that "codec provider" setting will ignore these mclk but still we use for reference. Also not sure about the ignorance of this params in config structure because BT-ssp-config still config this and followed the similar approach for HDMI-In ssp config.

io_clk			$BT_MCLK
		Object.Base.hw_config.1 {
			id		0
			name		"BT-SCO-WB"
			mclk_freq	$BT_MCLK

Please suggest if its really not need and would require to remove.

Reply:
@plbossart : Could you help to close this PR.

Reply:
@CBala21 I can close, but do you mean merge or do you have another PR ready for this feature ?

Reply:
@lgirdwood : I mean to review our comments and help to merge it on mainline.